[Windows] ctfe_init container "/bin/sh: 1: /root/logid.sh: not found"

EDIT: this issue was originally reported with the name: *MySql container has (fatal) incorrect Schema Error on startup: [ERROR] Native table 'performance_schema' ... has the wrong structure*. I've since change the issue name to one that I believe a user would encounter before the error I originally reported. 

**Description**

<!--Tell us about the bug. What went wrong? How can we reproduce it? What should the correct behavior be?
-->

Steps to reproduce:
1. clone fulcio
2. checkout tag `v1.4.4` (same error occurred on main on commit 7e4718f2d4d98d76f3b55c6ae6b895506101000b)
3. in powershell, run `docker compose up`

Expected:
all containers in docker compose up remain running. 

Actual:
MySql container does not start up properly on `docker compose up`. 
There are two errors that occur: 
```
[ERROR] Missing system table mysql.proxies_priv; please run mysql_upgrade to create it
[ERROR] Native table 'performance_schema'.'events_waits_summary_by_thread_by_event_name' has the wrong structure
```
This causes other containers that depend on the mysql server to shutdown. 

I exec'd into the container to try `mysql_upgrade` and got the following:
```
PS ...\fulcio > docker exec -it fulcio-mysql-1 /bin/bash
root@9e5e2bb8a82f:/# mysql_upgrade
mysql_upgrade: Got error: 1045: Access denied for user 'root'@'localhost' (using password: NO) while connecting to the MySQL server
Upgrade process encountered error and will not continue.
root@9e5e2bb8a82f:/# sudo mysql_upgrade
bash: sudo: command not found
```

**Version**
Fulcio repo at: `HEAD detached at v1.4.4`

*Environment*
This was run on windows 11. And docker desktop (info collected from about):
- version 4.27.2 (137060)
- Engine: 25.0.3
- Compose: v2.24.5-desktop.1
- Kubernetes: v1.29.1

Full log of `docker compose up`:
```
PS ...\fulcio > docker compose up
[+] Running 8/8
 ✔ Network fulcio_default                  Created                                                                                                                                                          0.1s
 ✔ Container fulcio-dex-idp-1              Created                                                                                                                                                          0.1s
 ✔ Container fulcio-mysql-1                Created                                                                                                                                                          0.1s
 ✔ Container fulcio-trillian-log-signer-1  Created                                                                                                                                                          0.1s
 ✔ Container fulcio-trillian-log-server-1  Created                                                                                                                                                          0.2s
 ✔ Container fulcio-fulcio-server-1        Created                                                                                                                                                          0.2s
 ✔ Container fulcio-ctfe_init-1            Created                                                                                                                                                          0.1s
 ✔ Container fulcio-ct_server-1            Created                                                                                                                                                          0.1s
Attaching to ct_server-1, ctfe_init-1, dex-idp-1, fulcio-server-1, mysql-1, trillian-log-server-1, trillian-log-signer-1
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config issuer: http://dex-idp:8888/auth"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config storage: memory"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config static client: Fulcio in Docker Compose"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config connector: https://any.valid.url/"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config response types accepted: [code]"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config skipping approval screen"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config signing keys expire after: 24h0m0s"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config id tokens valid for: 1m0s"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config auth requests valid for: 24h0m0s"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="config refresh tokens rotation enabled: true"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="keys expired, rotating"
mysql-1                | 2024-03-11T13:46:53.070351Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
mysql-1                | 2024-03-11T13:46:53.072982Z 0 [Note] mysqld (mysqld 5.7.32) starting as process 1 ...
mysql-1                | 2024-03-11T13:46:53.078097Z 0 [Note] InnoDB: PUNCH HOLE support available
mysql-1                | 2024-03-11T13:46:53.078551Z 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
mysql-1                | 2024-03-11T13:46:53.078743Z 0 [Note] InnoDB: Uses event mutexes
mysql-1                | 2024-03-11T13:46:53.078856Z 0 [Note] InnoDB: GCC builtin __atomic_thread_fence() is used for memory barrier
mysql-1                | 2024-03-11T13:46:53.078862Z 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
mysql-1                | 2024-03-11T13:46:53.078865Z 0 [Note] InnoDB: Using Linux native AIO
mysql-1                | 2024-03-11T13:46:53.080845Z 0 [Note] InnoDB: Number of pools: 1
mysql-1                | 2024-03-11T13:46:53.081008Z 0 [Note] InnoDB: Using CPU crc32 instructions
mysql-1                | 2024-03-11T13:46:53.083745Z 0 [Note] InnoDB: Initializing buffer pool, total size = 128M, instances = 1, chunk size = 128M
mysql-1                | 2024-03-11T13:46:53.095651Z 0 [Note] InnoDB: Completed initialization of buffer pool
mysql-1                | 2024-03-11T13:46:53.099590Z 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
mysql-1                | 2024-03-11T13:46:53.121469Z 0 [Note] InnoDB: Highest supported file format is Barracuda.
mysql-1                | 2024-03-11T13:46:53.143109Z 0 [Note] InnoDB: Creating shared tablespace for temporary tables
mysql-1                | 2024-03-11T13:46:53.143201Z 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
mysql-1                | 2024-03-11T13:46:53.174359Z 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
mysql-1                | 2024-03-11T13:46:53.175968Z 0 [Note] InnoDB: 96 redo rollback segment(s) found. 96 redo rollback segment(s) are active.
mysql-1                | 2024-03-11T13:46:53.176019Z 0 [Note] InnoDB: 32 non-redo rollback segment(s) are active.
mysql-1                | 2024-03-11T13:46:53.177839Z 0 [Note] InnoDB: Waiting for purge to start
mysql-1                | 2024-03-11T13:46:53.231919Z 0 [Note] InnoDB: 5.7.32 started; log sequence number 1306960
mysql-1                | 2024-03-11T13:46:53.235744Z 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
mysql-1                | 2024-03-11T13:46:53.236055Z 0 [Note] Plugin 'FEDERATED' is disabled.
mysql-1                | 2024-03-11T13:46:53.240342Z 0 [Note] InnoDB: Buffer pool(s) load completed at 240311 13:46:53
mysql-1                | 2024-03-11T13:46:53.243575Z 0 [Note] Found ca.pem, server-cert.pem and server-key.pem in data directory. Trying to enable SSL support using them.
mysql-1                | 2024-03-11T13:46:53.243617Z 0 [Note] Skipping generation of SSL certificates as certificate files are present in data directory.
mysql-1                | 2024-03-11T13:46:53.244962Z 0 [Warning] CA certificate ca.pem is self signed.
mysql-1                | 2024-03-11T13:46:53.245004Z 0 [Note] Skipping generation of RSA key pair as key files are present in data directory.
mysql-1                | 2024-03-11T13:46:53.245894Z 0 [Note] Server hostname (bind-address): '*'; port: 3306
mysql-1                | 2024-03-11T13:46:53.246231Z 0 [Note] IPv6 is available.
mysql-1                | 2024-03-11T13:46:53.246384Z 0 [Note]   - '::' resolves to '::';
mysql-1                | 2024-03-11T13:46:53.246414Z 0 [Note] Server socket created on IP: '::'.
mysql-1                | 2024-03-11T13:46:53.257861Z 0 [Warning] Insecure configuration for --pid-file: Location '/var/run/mysqld' in the path is accessible to all OS users. Consider choosing a different directory.
mysql-1                | 2024-03-11T13:46:53.259341Z 0 [ERROR] Missing system table mysql.proxies_priv; please run mysql_upgrade to create it
mysql-1                | 2024-03-11T13:46:53.262548Z 0 [ERROR] Native table 'performance_schema'.'events_waits_summary_by_thread_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262585Z 0 [ERROR] Native table 'performance_schema'.'events_waits_summary_by_user_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262606Z 0 [ERROR] Native table 'performance_schema'.'events_waits_summary_by_account_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262625Z 0 [ERROR] Native table 'performance_schema'.'events_waits_summary_global_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262643Z 0 [ERROR] Native table 'performance_schema'.'file_instances' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262663Z 0 [ERROR] Native table 'performance_schema'.'file_summary_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262681Z 0 [ERROR] Native table 'performance_schema'.'file_summary_by_instance' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262697Z 0 [ERROR] Native table 'performance_schema'.'host_cache' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262715Z 0 [ERROR] Native table 'performance_schema'.'mutex_instances' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262734Z 0 [ERROR] Native table 'performance_schema'.'objects_summary_global_by_type' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262751Z 0 [ERROR] Native table 'performance_schema'.'performance_timers' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262801Z 0 [ERROR] Native table 'performance_schema'.'rwlock_instances' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262825Z 0 [ERROR] Native table 'performance_schema'.'setup_actors' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262841Z 0 [ERROR] Native table 'performance_schema'.'setup_consumers' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262859Z 0 [ERROR] Native table 'performance_schema'.'setup_instruments' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262875Z 0 [ERROR] Native table 'performance_schema'.'setup_objects' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262890Z 0 [ERROR] Native table 'performance_schema'.'setup_timers' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262909Z 0 [ERROR] Native table 'performance_schema'.'table_io_waits_summary_by_index_usage' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262926Z 0 [ERROR] Native table 'performance_schema'.'table_io_waits_summary_by_table' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262943Z 0 [ERROR] Native table 'performance_schema'.'table_lock_waits_summary_by_table' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262958Z 0 [ERROR] Native table 'performance_schema'.'threads' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262975Z 0 [ERROR] Native table 'performance_schema'.'events_stages_current' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.262991Z 0 [ERROR] Native table 'performance_schema'.'events_stages_history' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263008Z 0 [ERROR] Native table 'performance_schema'.'events_stages_history_long' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263025Z 0 [ERROR] Native table 'performance_schema'.'events_stages_summary_by_thread_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263043Z 0 [ERROR] Native table 'performance_schema'.'events_stages_summary_by_account_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263092Z 0 [ERROR] Native table 'performance_schema'.'events_stages_summary_by_user_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263115Z 0 [ERROR] Native table 'performance_schema'.'events_stages_summary_by_host_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263135Z 0 [ERROR] Native table 'performance_schema'.'events_stages_summary_global_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263151Z 0 [ERROR] Native table 'performance_schema'.'events_statements_current' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263168Z 0 [ERROR] Native table 'performance_schema'.'events_statements_history' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263187Z 0 [ERROR] Native table 'performance_schema'.'events_statements_history_long' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263206Z 0 [ERROR] Native table 'performance_schema'.'events_statements_summary_by_thread_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263226Z 0 [ERROR] Native table 'performance_schema'.'events_statements_summary_by_account_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263244Z 0 [ERROR] Native table 'performance_schema'.'events_statements_summary_by_user_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263262Z 0 [ERROR] Native table 'performance_schema'.'events_statements_summary_by_host_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263300Z 0 [ERROR] Native table 'performance_schema'.'events_statements_summary_global_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263321Z 0 [ERROR] Native table 'performance_schema'.'events_statements_summary_by_digest' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263340Z 0 [ERROR] Native table 'performance_schema'.'events_statements_summary_by_program' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263356Z 0 [ERROR] Native table 'performance_schema'.'events_transactions_current' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263371Z 0 [ERROR] Native table 'performance_schema'.'events_transactions_history' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263389Z 0 [ERROR] Native table 'performance_schema'.'events_transactions_history_long' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263407Z 0 [ERROR] Native table 'performance_schema'.'events_transactions_summary_by_thread_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263429Z 0 [ERROR] Native table 'performance_schema'.'events_transactions_summary_by_account_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263447Z 0 [ERROR] Native table 'performance_schema'.'events_transactions_summary_by_user_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263464Z 0 [ERROR] Native table 'performance_schema'.'events_transactions_summary_by_host_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263481Z 0 [ERROR] Native table 'performance_schema'.'events_transactions_summary_global_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263497Z 0 [ERROR] Native table 'performance_schema'.'users' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263514Z 0 [ERROR] Native table 'performance_schema'.'accounts' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263530Z 0 [ERROR] Native table 'performance_schema'.'hosts' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263551Z 0 [ERROR] Native table 'performance_schema'.'socket_instances' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263571Z 0 [ERROR] Native table 'performance_schema'.'socket_summary_by_instance' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263590Z 0 [ERROR] Native table 'performance_schema'.'socket_summary_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263610Z 0 [ERROR] Native table 'performance_schema'.'session_connect_attrs' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263629Z 0 [ERROR] Native table 'performance_schema'.'session_account_connect_attrs' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263649Z 0 [ERROR] Native table 'performance_schema'.'memory_summary_global_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263666Z 0 [ERROR] Native table 'performance_schema'.'memory_summary_by_account_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263686Z 0 [ERROR] Native table 'performance_schema'.'memory_summary_by_host_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263706Z 0 [ERROR] Native table 'performance_schema'.'memory_summary_by_thread_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263725Z 0 [ERROR] Native table 'performance_schema'.'memory_summary_by_user_by_event_name' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263744Z 0 [ERROR] Native table 'performance_schema'.'table_handles' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263762Z 0 [ERROR] Native table 'performance_schema'.'metadata_locks' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263802Z 0 [ERROR] Native table 'performance_schema'.'replication_connection_configuration' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263827Z 0 [ERROR] Native table 'performance_schema'.'replication_group_members' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263846Z 0 [ERROR] Native table 'performance_schema'.'replication_connection_status' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263863Z 0 [ERROR] Native table 'performance_schema'.'replication_applier_configuration' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263879Z 0 [ERROR] Native table 'performance_schema'.'replication_applier_status' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263897Z 0 [ERROR] Native table 'performance_schema'.'replication_applier_status_by_coordinator' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263913Z 0 [ERROR] Native table 'performance_schema'.'replication_applier_status_by_worker' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263930Z 0 [ERROR] Native table 'performance_schema'.'replication_group_member_stats' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263946Z 0 [ERROR] Native table 'performance_schema'.'prepared_statements_instances' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263962Z 0 [ERROR] Native table 'performance_schema'.'user_variables_by_thread' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263977Z 0 [ERROR] Native table 'performance_schema'.'status_by_account' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.263992Z 0 [ERROR] Native table 'performance_schema'.'status_by_host' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.264007Z 0 [ERROR] Native table 'performance_schema'.'status_by_thread' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.264023Z 0 [ERROR] Native table 'performance_schema'.'status_by_user' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.264039Z 0 [ERROR] Native table 'performance_schema'.'global_status' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.264054Z 0 [ERROR] Native table 'performance_schema'.'session_status' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.264070Z 0 [ERROR] Native table 'performance_schema'.'variables_by_thread' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.264085Z 0 [ERROR] Native table 'performance_schema'.'global_variables' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.264100Z 0 [ERROR] Native table 'performance_schema'.'session_variables' has the wrong structure
mysql-1                | 2024-03-11T13:46:53.264681Z 0 [Note] Event Scheduler: Loaded 0 events
mysql-1                | 2024-03-11T13:46:53.264965Z 0 [Note] mysqld: ready for connections.
mysql-1                | Version: '5.7.32'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
trillian-log-signer-1  | I0311 13:46:53.505132       1 main.go:108] **** Log Signer Starting ****
trillian-log-signer-1  | W0311 13:46:53.507074       1 tree_storage.go:89] Failed to set strict mode on mysql db: Error 1130: Host '172.20.0.4' is not allowed to connect to this MySQL server
trillian-log-signer-1  | F0311 13:46:53.507140       1 main.go:115] Failed to get storage provider: Error 1130: Host '172.20.0.4' is not allowed to connect to this MySQL server
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="keys rotated, next rotation: 2024-03-12 13:46:53.551215009 +0000 UTC"
dex-idp-1              | time="2024-03-11T13:46:53Z" level=info msg="listening (http) on 0.0.0.0:8888"
trillian-log-server-1  | I0311 13:46:53.615355       1 main.go:97] **** Log Server Starting ****
trillian-log-server-1  | W0311 13:46:53.617029       1 tree_storage.go:89] Failed to set strict mode on mysql db: Error 1130: Host '172.20.0.5' is not allowed to connect to this MySQL server
trillian-log-server-1  | F0311 13:46:53.617090       1 main.go:118] Failed to get storage provider: Error 1130: Host '172.20.0.5' is not allowed to connect to this MySQL server
fulcio-server-1        | 2024-03-11T13:46:54.619Z       INFO    app/grpc.go:214 listening on grpc at [::]:5554
fulcio-server-1        | 2024-03-11T13:46:54.619Z       INFO    app/grpc.go:262 listening on grpc at @fulcio-legacy-grpc-socket
fulcio-server-1        | 2024-03-11T13:46:54.621Z       INFO    app/http.go:111 listening on http at 0.0.0.0:5555
ctfe_init-1            | /bin/sh: 1: /root/logid.sh: not found
trillian-log-signer-1 exited with code 0
Gracefully stopping... (press Ctrl+C again to force)
service "ctfe_init" didn't complete successfully: exit 127
```

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,1 +1,3 @@
 /pkg/generated/protobuf/** linguist-generated
+
+*.sh text eol=lf

